com.unity.ml-agents/Runtime/Communicator/RpcCommunicator.cs

```diff
@@ -114,10 +115,12 @@ public UnityRLInitParameters Initialize(CommunicatorInitParameters initParameter
                     },
                     out input);
 
-                var pythonCommunicationVersion = initializationInput.RlInitializationInput.CommunicationVersion;
                 var pythonPackageVersion = initializationInput.RlInitializationInput.PackageVersion;
+                var pythonCommunicationVersion = initializationInput.RlInitializationInput.CommunicationVersion;
                 var unityCommunicationVersion = initParameters.unityCommunicationVersion;
 
+                TrainingAnalytics.SetTrainerInformation(pythonPackageVersion, pythonCommunicationVersion);
```

Comment:
I don't love this, but it seems liked the best way to push this info to the analytics code for later use. We could potentially "pull" it from here in analytics, but that seems more brittle.